examples/favorites_advanced/getx_favorites_advanced/lib/base/common_bindings/home_binding.dart

```diff
@@ -8,5 +13,11 @@ class HomeBinding implements Bindings {
     Get.put(NavbarController());
     // ignore: cascade_invocations
     Get.put(FavoritePuppiesController());
+    // ignore: cascade_invocations
+    Get.put(PuppiesRepository(ImagePicker()));
```

Comment:
@RomanovaPrime is there are reason to not use cascade invocation

Reply:
No, there isn't. I use it now.